src/Markdown.js

```diff
@@ -56,23 +57,47 @@ export default class Markdown {
         return is_plain;
     }
 
-    toHTML() {
+    render(html) {
```

Comment:
Please don't call this `render`, as it is confusing and can make readers mistake the `Markdown` class as a React class.